app/src/main/java/it/niedermann/nextcloud/deck/util/ColorUtil.java

```diff
@@ -34,6 +34,31 @@ else if (isColorDark(color))
         return ret;
     }
 
+    /**
+     * @return well formated #123456 color string
+     */
+    public static String getCleanHexaColorString(String input) {
+        if (input == null) {
+            throw new IllegalArgumentException();
+        }
+        switch (input.length()) {
+            case 3:
+                return '#' + input + input;
```

Comment:
Hi there,

Here is a mistake. For the color `#af0` a correct value is `#aaff00`. But this line would derive `#af0af0` which is a wrong color in comparison to the origin one.